cmd/repl/repl.go

```diff
@@ -97,15 +115,15 @@ func main() {
 		}
 	}
 
-	for {
-		fmt.Print("sqlflow> ")
-		slct := readStmt()
-		fmt.Println("")
-
+	isTerminal := !flagPassed("execute", "e", "file", "f") && terminal.IsTerminal(syscall.Stdin)
+	sqlRun := func(stmt string) {
```

Comment:
I propose to separate `sqlRun` out from `main` to be a function definition as the following

```go
func runStmt(stmt string, isTerminal bool)
```

so to make `main` shorter and more comprehensive.

Reply:
Done.